src/components/interview.ts

```diff
@@ -103,6 +103,18 @@ export const getInterviewers = async (
   return res;
 };
 
+export const getInterviewerDomainsString = async (id: string): Promise<string> => {
+  const userDomains = await getDomains(id);
+  if (userDomains.length === 0) {
```

Comment:
Is it possible to reuse the `getDomainsString` helper? It's defined near the top of this file.

Reply:
Yes, sorry about that, didn't notice the function, I've changed it.

Reply:
No need to apologize! I'll approve the PR once you push the new changes.